src/Microsoft.Extensions.Http/DefaultHttpClientFactory.cs

```diff
@@ -128,6 +128,21 @@ public HttpClient CreateClient(string name)
             return client;
         }
 
+        public HttpMessageHandler CreateHandler(string name)
+        {
+            if (name == null)
+            {
+                throw new ArgumentNullException(nameof(name));
+            }
+
+            var entry = _activeHandlers.GetOrAdd(name, _entryFactory).Value;
+            var handler = new LifetimeTrackingHttpMessageHandlerWrapper(entry.Handler);
+
+            StartHandlerEntryTimer(entry);
+
+            return handler;
```

Comment:
Should `CreateClient` call into this?

Reply:
Good point. I think I just did a minimal "don't touch existing stuff" approach as a first pass.